benchmarks/benchmark_throughput_async.py

```diff
@@ -0,0 +1,480 @@
+"""Benchmark offline inference throughput."""
```

Comment:
Can we just improve `benchmark_throughput.py` to support async? Seems like the only difference is using `uvloop` to run the async function.

Reply:
yeah np

Reply:
Yep that was the plan all along, now done in https://github.com/vllm-project/vllm/pull/7964.